packages/demos/src/history-service/history-consumer.tsx

```diff
@@ -6,7 +6,7 @@ import {
   InputGroup,
   Label
 } from '@blueprintjs/core';
-import {History} from '@feature-hub/history-service';
+import {History} from 'history';
```

Comment:
Should we add `@types/history` to the dependencies of `demos`?

Reply:
done